plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestPostgreSqlPlugin.java

```diff
@@ -34,4 +53,97 @@ public void testCreateConnector()
                         "bootstrap.quiet", "true"),
                 new TestingPostgreSqlConnectorContext()).shutdown();
     }
+
+    @Test
+    void testUnknownPropertiesAreRedactable()
```

Comment:
`testUnknownPropertiesAreRedactable` -> `testUnknownPropertiesAreSecuritySensitive` (if you decide to change the SPI method name). Here and elsewhere.